src/index.html

```diff
@@ -1,13 +1,401 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.png"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a class="icon icon--menu-opener" href="#menu"></a>
+
+            <a href="#" class="logo header__logo">
+              <img
+                class="logo__image"
+                src="./images/logo.png"
+                alt="Maimi Condo Kings"
+              >
+            </a>
+
+            <a href="#" class="icon icon--phone header__call"></a>
```

Comment:
You should be able to call, clicking on this link. Change href value to `tel: phone-number`.